{"name": "Проверка логина курьера с неверными login и password", "status": "passed", "steps": [{"name": "Логин курьера", "status": "passed", "steps": [{"name": "Получение логина курьера", "status": "passed", "start": 1731160693893, "stop": 1731160693893}, {"name": "Получение пароля курьера", "status": "passed", "start": 1731160693893, "stop": 1731160693893}], "parameters": [{"name": "login", "value": "''"}, {"name": "password", "value": "''"}], "start": 1731160693893, "stop": 1731160694182}], "parameters": [{"name": "key", "value": "'login'"}], "start": 1731160693893, "stop": 1731160694182, "uuid": "010fd759-0c6c-42c8-a9e2-fb7bc546bfb9", "historyId": "f7cb2c504466eb75df57dfa946b09aa6", "testCaseId": "aba4e33e6ffafaa96d0ee9f3ab7257b9", "fullName": "tests.test_the_courier.TestLoginCourier#test_login_with_wrong_credentials_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_the_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "HP-VICTUS-V"}, {"name": "thread", "value": "21444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_the_courier"}]}